Src/Support/GoogleApis.Auth.PlatformServices_Shared/OAuth2/GoogleCredential.cs

```diff
@@ -158,8 +158,10 @@ Task<string> ITokenAccess.GetAccessTokenForRequestAsync(string authUri, Cancella
             return credential.GetAccessTokenForRequestAsync(authUri, cancellationToken);
         }
 
-        /// <summary>Provides access to the underlying credential object</summary>
-        internal ICredential UnderlyingCredential { get { return credential; } }
+        /// <summary>
+        /// Gets the underlying credential instance being wrapped.
+        /// </summary>
+        public ICredential UnderlyingCredential => credential;
```

Comment:
Thanks for the explanation. It does sound as if making this public is the cleanest way to go.